ui/src/infra-model/tabs/tabs.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import TabNavItem from 'infra-model/tabs/tab-nav-item';
+import TabContent from 'infra-model/tabs/tab-content';
+import { useCommonDataAppSelector } from 'store/hooks';
+import { InfraModelListContainer } from 'infra-model/list/infra-model-list-container';
+import { useTranslation } from 'react-i18next';
+import { commonActionCreators, InfraModelTabType } from 'common/common-slice';
+import { createDelegates } from 'store/store-utils';
+import { VelhoFileList } from 'infra-model/velho/velho-file-list';
+
+export type TabsProps = {
+    activeTab: InfraModelTabType;
+};
+
+const Tabs: React.FC<TabsProps> = ({ activeTab }) => {
+    const { t } = useTranslation();
+    const changeTimes = useCommonDataAppSelector((state) => state.changeTimes);
+    const delegates = createDelegates(commonActionCreators);
+
+    //erottele omiin komponentteihin 1) tab-navigointi 2) tabeja vastaavat sisällöt?
```

Comment:
Jos/kun tämä pysyy näin simppelinä, niin mielestäni tämä voi olla hyvä nykymuodossaankin